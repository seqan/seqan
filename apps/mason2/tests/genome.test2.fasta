>1
CTGTAACTAAACACCGCTGTCCGGACTAAGGTCCGCATATTGTTCAGGTTTTATACAATGAGCGTCGAGC
CAGTTCAATGTGGCCCTAAACCTACTCAAGAAGTATCCCGAGGGATGCGAAGCTGCCGATGAGAGTTAGT
TGACATTCCGAGTCCGTGGATGGGTCCTCCTTCTTAGCGAATTACCGTCCATTTGCATGACGGATGCCTT
GCACTTGGTAACAGTAGCAATGTATAGGAAACAAAGAATCGAGTTCAACAGGTCGCATTCAGTGCCTTGC
GCACAAATATATAGGAGGAAACTCGCAACAGAATGCTATGACCAGTTTATAGAACTGTGCCGTAGACGGA
GTGCCAATCAGAAATAACCATAATGGGTTGCCAGGGTGTATGACAGGTTCTTGATAGTCCCAGAGCTTTC
CGTAACAAAGGAACTTTCACAACGATTGCCTCGTTATATTCCGCTGCACTGCATGACATTCATTGGAATG
GAAACAGCTGATTTCCTAGATAGCGCCGGGTATATTGTTACCGCCCCACACGCAGGCATCGGACCACATA
TTTTGGCACCTGGTTAGAGAGCTAGACTCGATCGAATAACTTTAGGGAAGCTCCATGTCATGTCCAACAC
CCGAAGAGTACAGGACATGTCAAAGGGAGGATTAAAGTTGACCCTATCGATGTATGTCAAGGGTGCTGAG
TTGCCCCAGGCTTGCGTATCCTACTGGCTTGATTGGGTACCTAGAGGTACCGTTGCGCATCGGCTCACAA
CGTTCAATGCAGTCCCAAGATACGGTGTGATGTCAGGACCTAAGCGACCCATGATCGCTGCTTCCTTGGA
AGAGAGAGCGACATTTCCGTCTCGCGCATTCTTTAGTAAGGGAGGAACTTTTTGTGAAAGATGGTGTTGG
GCTTCGAGACATTTATCGTGCCTTCTGGCTTACTCAGATTACGCTGAAGCCTTTGGGCCTGCCCATGACT
GTTGCAATCATTTGTCTGAA
>2
ACACCAATGACTTAACGAATGCCCACAAGACTGAAGGTTATCGATCTTGGAGAAAACTCCTGATTCTATA
GGGACCATCTCAGGCCCGACAATCCTTAAC
