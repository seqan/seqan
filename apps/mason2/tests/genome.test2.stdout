MASON GENOME SIMULATOR
======================

__OPTIONS____________________________________________________________________

VERBOSITY  	1

SEED       	1

OUTPUT FILE	genome.test2.fasta
CONTIG LENS	1000, 100

__SIMULATING GENOME__________________________________________________________

